### DISCUSSION TITLE
Specifying Extra Details and Hints

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#92] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/usage/extra-details-hints

**Comment by Studere01** [comment]
Thank you

**Comment by phillipeluiz** [comment]
Is there any way to insert labels?

**Comment by Nexustarget99** [comment]
The second option works great for embedded blocks, but I get an error randomly where it seems like the embed is being added multiple times to a card when I use the same embed for multiple forward cards.

**Comment by Stonesitter** [comment]
Is there a way to show images in the extras? I could not make it work with icremental multiline cards.

**Comment by anonlinekook** [comment]
I am not sure if I have accidentally changed a setting somewhere, but if I make a card and then put the extra stuff below it, when I click out of the block, the extra text automatically goes to the top of the block. Can anyone help me out with this? I would prefer if it were at the bottom.

ie:
<img width="680" alt="image" src="https://github.com/user-attachments/assets/5b8b8748-4c9d-4819-a5df-d4414e9520e4">

Then when I click out of the block
<img width="671" alt="image" src="https://github.com/user-attachments/assets/12481acd-c541-45a0-b55c-6bdbdc4cecd0">

If I then click back in to the block the extra text is now formatted to be above even though I had initially typed it below. ie:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/0b412769-d387-4325-8e5f-8a59bd9c1ee3">


Thank you so much for the anki plug-in, I am new to this but suspect this will be the future for me!

**Comment by sa82trip** [comment]
Hello :) 
Is it possible to collapse extra details by default?
It would be better if I can configure to collapse `Extra Details` when I flip a card.
